vxdev/update-code.sh

```diff
@@ -49,8 +53,26 @@ elif [[ $BRANCH == 'stable' ]]; then
 	cd vxsuite
 	./script/setup-dev
 	cd ..
+elif [[ $BRANCH == 'custom' ]]; then
+	read -p "Enter the branch name: " BRANCH_NAME
+	cd vxsuite
+	git checkout main
+	git pull
+	if [ !`git branch -r --list origin/$BRANCH_NAME` ]
+	then
+		echo "Invalid Branch Name. Closing in 3 seconds."
+		echo $BRANCH_NAME
+		exit 0
+	fi
+	git checkout $BRANCH_NAME
+	./script/setup-dev
+	cd ../kiosk-browser
+	git checkout main
+	git pull
+	cd ..
 fi
 
+exit 0
```

Comment:
Intentional?

```suggestion
```

Reply:
lol oops, just put this in for testing will remove